admin_manual/configuration/server/caching_configuration.rst

```diff
@@ -1,267 +1,402 @@
-| cache is configured, but the PHP module     | will be a full page issue, and the application will not be       |
-| missing.                                    | usable                                                           |
-+---------------------------------------------+------------------------------------------------------------------+
-| All enabled, but the Redis server is not    | The application will be usable. But any file operation will      |
-| running                                     | return a **"500 Redis went away"** exception                     |
-+---------------------------------------------+------------------------------------------------------------------+
-| If Memcache is configured for "local" and   | There will be a white page and an exception written to the logs, |
-| "distributed", but the class is missing     | This is because autoloading needs the missing class. So there is |
-|                                             | no way to show a page                                            |
-+---------------------------------------------+------------------------------------------------------------------+
+The caching backends supported by ownCloud are:
+
+* :ref:`APCu <apcu-label>`: This is a local cache for systems running PHP 5.6 and up. APCu 4.0.6 and up is required. Alternatively you can use `the Zend OPCache`. However, **it is not a data cache**, only an opcode cache.
+* :ref:`Redis <redis-label>`: This is a distributed cache for multi-server ownCloud installations. Version 2.2.6 or higher of the PHP Redis extension is required.
+* :ref:`Memcached <memcached-label>`: This is a distributed cache for multi-server ownCloud installations.
+
+.. note:: 
+   You may use *both* a local and a distributed cache. 
+   The recommended ownCloud caches are APCu and Redis. 
+
+.. note::
+   If you do not install and enable a local memory cache you will see a warning on your ownCloud admin page. A memory cache is not required and you may safely ignore the warning if you prefer.
+   If you enable only a distributed cache in your ``config.php`` (``memcache.distributed``) and not a local cache (``memcache.local``) you will still see the cache warning.
+ 
+Cache Directory Location
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The cache directory defaults to ``data/$user/cache`` where ``$user`` is the current user. 
+You may use the ``'cache_path'`` directive in ``config.php`` (See :doc:`config_sample_php_parameters`) to select a different location.
+    
+Cache Types
+-----------
+    
+.. _apcu-label:
     
 APCu
-----
+~~~~
+
+PHP 5.6 and up include the Zend OPcache in core, and on most Linux distributions it is enabled by default. 
+However, it *does not* bundle a data cache. 
+Given that, we recommend that you use APCu instead. APCu is a data cache *and* is available in most Linux distributions. 
+
+Installing APCu
+^^^^^^^^^^^^^^^
+
+.. code-block:: console
+   
+   # On RedHat/CentOS/Fedora systems running PHP 5.6 and up 
```

Comment:
I would say: on RedHat/CentOS/Fedora systems running PHP 7.0 since the package `rh-php70-php-devel` is for PHP 7.0 only